packages/components/src/input/src/Input.css

```diff
@@ -34,11 +34,20 @@
     width: 100%;
 }
 
+/* SELECTION */
 .o-ui-input input::selection,
 .o-ui-input textarea::selection {
     background-color: var(--o-ui-bg-alias-input-selection);
 }
 
+/* AUTOFILL */
+input:-webkit-autofill,
+textarea:-webkit-autofill,
+select:-webkit-autofill {
+    -webkit-box-shadow: 0 0 0 1000px var(--o-ui-bg-alias-accent-faint) inset !important;
+    -webkit-text-fill-color: var(--o-ui-text-alias-primary) !important;
+}
```

Comment:
Your selectors are too generic, you should only match orbit elements meaning, `.o-ui-input` selectors.

Reply:
Totally, copy pasta gone wrong

Reply:
Also, it's not a big deal but, adding code for "select" in this file is not great from a maintenance standpoint. This file is scope to text inputs.